File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int height = 0;
    do{
        height = get_int("Height:");
    }while(height < 1 || height > 8);
    int i = 1;
    int j = 0;

    while(i <= height){
        for(j = i; j < height;j++){
            printf(" ");
        }
        for(int k = 0; k < i;k++){
            printf("#");
        }
        printf("\n");
        i++;
    }

}

File: pset6/less/mario.py
from cs50 import get_int

l = 1
height = get_int("Height:")

while(height > 8 or height < 1):
    height = get_int("Height:")

for i in range(height, 0, -1):
    for j in range(i-1):
        print(end=" ")
    for k in range(l):
        print("#", end="")
    print()
    l += 1

